src/flownet/network_model/_generate_connections.py

```diff
@@ -453,7 +453,7 @@ def create_connections(
     aquifer_ends: List[Coordinate] = []
 
     aquifer_config = configuration.model_parameters.aquifer
-    if aquifer_config is not None:
+    if all(aquifer_config[0:3]) and all(aquifer_config.size_in_bulkvolumes):
```

Comment:
would it be an idea to add a configuration.flownet.aquifer boolean (like with fast_pyscal) so you can just have `if configuration.flownet.aquifer:` here?

Reply:
No, because that would mean you have to define the aquifer at two spots.

I fully agree with you that this if-statement is ugly. And I have now the same issue in other places as well.... I asked @markusdregi if there is a better solution for this.

Reply:
Do you really? Can't you just set it false as default, and set it to true if the parsing of the aquifer parameter passes all checks?

Reply:
There is no better solution for this as of now, but I've been considering for some time the possibility of supporting derived data, that cannot be configured by a user, but that will be computed based on the input configuration. That could fix your "problem" here.

Also, you can of course implement derived data in the current system (basically what Ole suggests above); but that will have the rather odd consequence of exposing a keyword to the user that the user will have no say over. She can flip it one way or the other, and it will make no difference as the schema will always set it to the correct value.